patches/coveo.analytics+2.30.38.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/node_modules/coveo.analytics/dist/browser.mjs b/node_modules/coveo.analytics/dist/browser.mjs
```

Comment:
Could we not fix this directly in the lib instead of patching it?

Reply:
Probably... but I prefer not venturing here now TBH.
In the end we all agree that its not so bleading edge to assume that `fetch` is now available globally in Node. And shouldn't need to be polyfilled.  So I guess that the lib could be modified to not "enforce" the polyfill, sure. But right now this seem quite out of scope for the matter at hand.